lib/inc/Recorder.h

```diff
@@ -290,6 +290,18 @@ namespace sairedis
                     _In_ sai_status_t status,
                     _In_ const uint64_t *count);
 
+            void recordQueryAattributeCapability(
```

Comment:
double "Aa" in name

Reply:
removed the extra "a" from the function name.